jscomp/others/jsxDOM.res

```diff
@@ -0,0 +1,621 @@
+  @as("aria-valuemin")
+  ariaValuemin?: float,
+  @as("aria-valuenow")
+  ariaValuenow?: float,
+  @as("aria-valuetext")
+  ariaValuetext?: string,
+  /* Live Region Attributes */
+  @as("aria-atomic")
+  ariaAtomic?: bool,
+  @as("aria-busy")
+  ariaBusy?: bool,
+  @as("aria-live") ariaLive?: [#off | #polite | #assertive | #rude],
+  @as("aria-relevant")
+  ariaRelevant?: string,
+  /* Drag-and-Drop Attributes */
+  @as("aria-dropeffect") ariaDropeffect?: [#copy | #move | #link | #execute | #popup | #none],
+  @as("aria-grabbed")
+  ariaGrabbed?: bool,
+  /* Relationship Attributes */
+  @as("aria-activedescendant")
+  ariaActivedescendant?: string,
+  @as("aria-colcount")
+  ariaColcount?: int,
+  @as("aria-colindex")
+  ariaColindex?: int,
+  @as("aria-colspan")
+  ariaColspan?: int,
+  @as("aria-controls")
+  ariaControls?: string,
+  @as("aria-describedby")
+  ariaDescribedby?: string,
+  @as("aria-errormessage")
+  ariaErrormessage?: string,
+  @as("aria-flowto")
+  ariaFlowto?: string,
+  @as("aria-labelledby")
+  ariaLabelledby?: string,
+  @as("aria-owns")
+  ariaOwns?: string,
+  @as("aria-posinset")
+  ariaPosinset?: int,
+  @as("aria-rowcount")
+  ariaRowcount?: int,
+  @as("aria-rowindex")
+  ariaRowindex?: int,
+  @as("aria-rowspan")
+  ariaRowspan?: int,
+  @as("aria-setsize")
+  ariaSetsize?: int,
+  /* react textarea/input */
+  defaultChecked?: bool,
+  defaultValue?: string,
+  /* global html attributes */
+  accessKey?: string,
+  className?: string /* substitute for "class" */,
+  contentEditable?: bool,
+  contextMenu?: string,
+  @as("data-testid") dataTestId?: string,
+  dir?: string /* "ltr", "rtl" or "auto" */,
+  draggable?: bool,
+  hidden?: bool,
+  id?: string,
+  lang?: string,
+  role?: string /* ARIA role */,
+  style?: style,
+  spellCheck?: bool,
+  tabIndex?: int,
+  title?: string,
+  /* html5 microdata */
+  itemID?: string,
+  itemProp?: string,
+  itemRef?: string,
+  itemScope?: bool,
+  itemType?: string /* uri */,
+  /* tag-specific html attributes */
+  accept?: string,
+  acceptCharset?: string,
+  action?: string /* uri */,
+  allowFullScreen?: bool,
+  alt?: string,
+  async?: bool,
+  autoComplete?: string /* has a fixed, but large-ish, set of possible values */,
+  autoCapitalize?: string /* Mobile Safari specific */,
+  autoFocus?: bool,
+  autoPlay?: bool,
+  challenge?: string,
+  charSet?: string,
+  checked?: bool,
+  cite?: string /* uri */,
+  crossOrigin?: string /* anonymous, use-credentials */,
+  cols?: int,
+  colSpan?: int,
+  content?: string,
+  controls?: bool,
+  coords?: string /* set of values specifying the coordinates of a region */,
+  data?: string /* uri */,
+  dateTime?: string /* "valid date string with optional time" */,
+  default?: bool,
+  defer?: bool,
+  disabled?: bool,
+  download?: string /* should really be either a boolean, signifying presence, or a string */,
+  encType?: string /* "application/x-www-form-urlencoded", "multipart/form-data" or "text/plain" */,
```

Comment:
Why not have an enum here?

`#"text/plain"` etc. work fine in ReScript syntax.